yt_dlp/options.py

```diff
@@ -410,12 +410,19 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
             '("-" for stdin). Can be used multiple times and inside other configuration files'))
     general.add_option(
         '--plugin-dirs',
-        dest='plugin_dirs', metavar='PATH', action='append',
+        metavar='PATH',
+        dest='plugin_dirs',
+        action='append',
         help=(
             'Path to an additional directory to search for plugins. '
             'This option can be used multiple times to add multiple directories. '
-            'Note that this currently only works for extractor plugins; '
-            'postprocessor plugins can only be loaded from the default plugin directories'))
+            'Add "no-external" to disable searching default external plugin directories (outside of python environment)'))
+    general.add_option(
+        '--no-plugins',
+        dest='plugins_enabled',
+        action='store_false',
```

Comment:
This should empty plugin_dirs instead imo. This way, --no-plugins --plugin-dir xxx` can be used to disable default plugins but load custom ones

Reply:
With the current logic (with no-external), we need a way to differentiate no plugin dirs (only search PYTHON_PATH) and do not load plugins at all (searching and loading skipped, etc.).

If we keep no-external, we could add another option for python_path?

> This way, --no-plugins --plugin-dir xxx can be used to disable default plugins but load custom ones

My original idea was `--no-plugins` cancels out any `--plugin-dirs` added

Reply:
`--config a --no-config --config b` only loads `b`. I'm suggesting `--plugin-locations` should behave the same